### DISCUSSION TITLE
Add Nightly build to Chocolatey

### DISCUSSION THREAD

**Discussion by Infinitay** [Chatterino/chatterino2#3073] [Ideas] [answered]
Seems like we already have beta builds being deployed to Chocolatey. Could we also **deploy** **nightly** builds with latest commits to **Chocolatey** as well?

Personally I primarily used the nightly builds until I realized Chatterino was on Chocolatey. I swapped over to installing Chatterino that way because it was easier than constantly downloading the nightly build and extracting the update to my current install location. Would be nice if I was able to be able to use nightly builds via Chocolatey and keep the streamlined install and updates.

**Comment by Felanbird** [comment]
cc: @ALazyMeme

**Comment by ALazyMeme** [comment, marked as answer]
Builds on chocolatey are released manually in a fortnightly fashion; I was forced to hide 2 builds due to the funky 2.3.3-beta release (then re-release). I don't have the time to do releases weekly and have no intention of weekly builds.

The biggest reason I chose fortnightly is due to there (quite often) not being enough features/fixes done in a week to justify my time on a release. Reminder that these builds aren't automatic, they're all manually done by myself.

**Reply by Infinitay** [reply]
> not being enough features/fixes done in a week
> they're all manually done by myself.

Ah alright fair enough. I was under the assumptions builds were done automatically with Actions.

**Reply by ALazyMeme** [reply]
Nah I haven't figured that one out yet. The release files of chatterino are...messy so it's been difficult for me to figure it out/be bothered to figure it ou.
